keanu-project/src/test/java/io/improbable/keanu/network/ProtobufTest.java

```diff
@@ -94,6 +94,23 @@ public void youCanSaveAndLoadANetworkWithValues() throws IOException {
         latentGaussianVertex.sample();
     }
 
+    @Test
```

Comment:
Do you need to add some tests for failure cases relating to shape? Similar to your other `loadFailsIf...` tests.

Reply:
There isn't really a failure case for shape - it can happily be left out entirely from the protobuf ( in fact that's what happens for scalars anyway).